adal/src/main/java/com/microsoft/aad/adal/SSOStateSerializer.java

```diff
@@ -62,15 +62,18 @@
             .registerTypeAdapter(TokenCacheItem.class, new TokenCacheItemSerializationAdapater())
             .create();
 
-    private int getVersion() {
-        return version;
+    /**
+     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+     */
+    @SuppressWarnings("unused")
```

Comment:
don't know why i missed it, but why do you remove the private for SSOStateSerializer?